d3a_api_client/utils.py

```diff
@@ -50,21 +52,17 @@ def post_request(endpoint, data, jwt_token):
         data=json.dumps(data),
         headers={"Content-Type": "application/json",
                  "Authorization": f"JWT {jwt_token}"})
-    if resp.status_code != 200:
-        logging.error(f"Request to {endpoint} failed with status code {resp.status_code}."
-                      f"Response body: {resp.text} {resp.reason}")
-        return False
-    return True
+    return json.loads(resp.text) if request_response_returns_http_200(endpoint, resp) else None
 
 
 def blocking_post_request(endpoint, data, jwt_token):
     data["transaction_id"] = str(uuid.uuid4())
-    response = requests.post(
+    resp = requests.post(
```

Comment:
Any reason to rename variable from `response ` to `resp ` ?

Reply:
To be consistent with the other functions in this module, too.